### PR TITLE
Add crude Keymaster health dashboard to keymaster-eventmon.

### PR DESCRIPTION
**Pull request by rgooch**

### CODE REVIEW

File: eventmon/monitord/impl.go [hunk 1]
```diff
@@ -100,18 +102,27 @@ func (m *Monitor) updateNotifierList(logger log.Logger) {
 		logger.Printf("Deleting old keymaster server: %s\n", addr)
 		m.closers[addr] <- struct{}{}
 		delete(m.closers, addr)
+		delete(m.keymasterStatus, addr)
```
Comment (cviecco) [inline eventmon/monitord/impl.go]: dont you need a lock for the keymasterstatus?

File: eventmon/monitord/impl.go [hunk 2]
```diff
@@ -221,6 +232,35 @@ func receiveV0(reader io.Reader) (eventmon.EventV0, error) {
 }
 
 func (m *Monitor) writeHtml(writer io.Writer) {
+	fmt.Fprintln(writer, "Keymasters:<br>")
+	fmt.Fprintln(writer, `<table border="1">`)
+	fmt.Fprintln(writer, "  <tr>")
+	fmt.Fprintln(writer, "    <th>IP Addr</th>")
+	fmt.Fprintln(writer, "    <th>Status</th>")
+	fmt.Fprintln(writer, "  </tr>")
+	m.mutex.RLock()
```
Comment (cviecco) [inline eventmon/monitord/impl.go]: Since you are using defers why not defer here after the lock?

File: eventmon/monitord/impl.go (hunk 1 above)
Comment (rgooch) [reply eventmon/monitord/impl.go]: Oops. Yeah. Fixed.

File: eventmon/monitord/impl.go (hunk 2 above)
Comment (rgooch) [reply eventmon/monitord/impl.go]: Yes. That's what I intended. I inserted the new code prior to the defer :-/

**Comment by cviecco** [comment]
LGTM now

**Pull request merged**